js/lightbeam.js

```diff
@@ -113,11 +113,25 @@ async function initLightBeam() {
   websites = await storeChild.getAll();
   renderGraph(websites);
 
+  const saveData = document.getElementById('save-data-button');
+  saveData.addEventListener('click', async function () {
+    const data = await storeChild.getAll();
+    const blob = new Blob([JSON.stringify(data ,' ' , 2)],
+      {type : 'application/json'});
+    const url = window.URL.createObjectURL(blob);
+    const a = document.createElement('a');
+    a.href = url;
+    a.download = 'lightbeamData.json';
+    a.target = '_blank';
+    document.body.appendChild(a);
+    a.click();
```

Comment:
@pauljt @jonathanKingston Why are we downloading by appending a link to the DOM and clicking it? Is this a common way (I honestly have never done this before)? Seems like the Web Extension downloads API would be the way to go.

Also when I try out your branch, Paul, the screen goes blank with a loading gif on the tab. This is the case when I remove `a.target='_blank'` as well.